Possible missing file history

I got the bellow error, it seems a history file is missing.

ERROR in ./entry/client/root.jsx
Module not found: Error: Cannot resolve module 'history' in C:\MeteorLab\redux-meteor\entry\client
 @ ./entry/client/root.jsx 13:15-33

--- a/webpack.packages.json
+++ b/webpack.packages.json
@@ -35,6 +35,7 @@
 
   "redux-devtools": "^3.0.1",
   "redux-devtools-log-monitor": "^1.0.2",
-  "redux-devtools-dock-monitor": "^1.0.1"
+  "redux-devtools-dock-monitor": "^1.0.1",
+  "history": "^2.0.0-rc2"
 
 }

